lib/CL/devices/common_utils.c

```diff
@@ -616,20 +630,37 @@ pocl_cpu_validate_khr_gemm (cl_bool TransA,
 }
 #endif
 
+/**
+ * CPU_DBK_TAG
+ */
 int
 pocl_cpu_supports_dbk (cl_device_id device,
                        BuiltinKernelId kernel_id,
                        const void *kernel_attributes)
 {
+  switch (kernel_id)
+    {
 #ifdef HAVE_LIBXSMM
-  /* the following code checks for LIBXSMM specific requirements on Tensors */
-  return pocl_validate_dbk_attributes (kernel_id, kernel_attributes,
-                                       pocl_cpu_validate_khr_gemm);
-#else
-  POCL_RETURN_ERROR_ON (1, CL_UNSUPPORTED_DBK,
-                        "The CPU driver must be compiled with libxsmm "
-                        "to support tensor DBKs\n");
+    case POCL_CDBI_DBK_EXP_GEMM:
+    case POCL_CDBI_DBK_EXP_MATMUL:
+      {
+        /* the following code checks for LIBXSMM specific requirements on
```

Comment:
Please use a full sentence.

Reply:
The formatting ended up putting tensors on a new line, but I can try to make the original comment more clear.